src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,112 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double GROWTH_RATE = 1.5;
+    private static final int MAX_ARRAY_SIZE = Integer.MAX_VALUE;
```

Comment:
You can use this constant Integer.MAX_VALUE directly. Don't need create constant for constant